### ISSUE TITLE
[Accessibility] Narrator cannot announce the checked status for ListView item when select the checkbox by mouse click 

### ISSUE THREAD

**Issue by John-Qiao** [dotnet/winforms#10366]
### .NET version

9.0.100-alpha.1.23564.26

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No, this issue also repros on .NET 6.0/7.0/8.0.

### Issue description

Narrator cannot announce the checked status for ListView item when select the checkbox by mouse click.

**The behavior for .NET 9.0:**
![Core1](https://github.com/dotnet/winforms/assets/45864985/1b4aba4f-f78a-4785-bbf5-a4fb55f7bd4a)

https://github.com/dotnet/winforms/assets/45864985/614a0708-75de-4cc3-9605-3349aba537ee

**The behavior for .Net framework:**
![Framework1](https://github.com/dotnet/winforms/assets/45864985/2c35f0de-a1c1-48b1-88d2-8463992adcf3)

### Steps to reproduce

1. Open the [ListView-Test.zip](https://github.com/dotnet/winforms/files/13447160/ListView-Test.zip) project.
2. Build and run project.
3. Launch Narrator tool, try to focus on the first item of the ListView with the mouse.
4. Click the item1's checkbox by left mouse click.

**Comment by Tanya-Solyanik** [comment]
Mouse click and narrator is a less important scenario, as chances that mouse user relies on the narrator are low. Does narrator work with keyboard navigation, @John-Qiao?

**Comment by merriemcgaw** [comment]
@John-Qiao if it does work with keyboard you can go ahead and close this issue

**Comment by John-Qiao** [comment]
@merriemcgaw @Tanya-Solyanik yes, it works with keyboard, so closed this issue.
